src/app/pages/home-page/components/faqs-section/faqs-section.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <div class="text-container">
+    <h1>{{ title }}</h1>
+    {{ description }}
+  </div>
+  <mat-expansion-panel *ngFor="let faq of faqs">
+    <mat-expansion-panel-header>
+      {{faq.question}}
+    </mat-expansion-panel-header>
+    <mat-panel-description>
+      {{faq.answer}}
+    </mat-panel-description>
+  </mat-expansion-panel>
+</div>
```

Comment:
You can add the title and the description here, at the top of the html

Reply:
Also, center all the content, make the title bigger and change the color and/or the opacity of the description.
Also make the FAQ panel blue with white text when someone hovers over it. 
You can also make the background light blue and put the panels in a white box. Check out how it looks on https://dscupt.tech/